jupyterlab/commands.py

```diff
@@ -568,6 +579,32 @@ def uninstall_extension(self, name):
         self.logger.warn('No labextension named "%s" installed' % name)
         return False
 
+    def update_extension(self, name=None, all_ext=False):
+        if all_ext:
+            return any([
```

Comment:
Do you intend for this to fail after the first failure or for it to continue despite failures and then report afterward that there were some failures? Its current behavior seems to be that if there is a failure, it will continue with the rest of the extensions, unless I am misunderstanding it.

Reply:
The return value here is to indicate whether a rebuild is recommended (will be performed automatically if recommended unless `--no-build` flag is given). For several packages, one package should be enough, that's why there's an `any` clause. Any exceptions during the install will currently bubble up and abort the entire operation I guess. I could try to expand this loop and add some exception handling here to allow for independent upgrading of the different extension.

Reply:
Because you are first creating the list (*i.e.* populating the `[]` which is inside `any()`) and then using `any`, every single item in the list is processed first, isn't it? So if one fails, it still continues?